lms/services/lti_outcomes.py

```diff
@@ -118,11 +50,19 @@ def read_result(self, outcomes_request_params):  # pylint:disable=no-self-use
 
         :return: The last-submitted score or `None` if no score has been submitted.
         """
-        body = READ_RESULT_REQUEST_TEMPLATE.render(
-            lis_result_sourcedid=outcomes_request_params.lis_result_sourcedid
-        )
```

Comment:
So previously both `read_result()` and `record_result()` had to render their respective templates themselves and pass the rendered body (an XML string) into `_send_request()`. Now they can both just pass in a dict literal :+1: